frontend/src/scenes/max/maxGlobalLogic.tsx

```diff
@@ -76,6 +76,21 @@ export const STATIC_TOOLS: ToolRegistration[] = [
         name: 'Query data',
         description: 'Query data by creating insights and SQL queries',
     },
+    {
+        identifier: 'create_survey' as const,
+        name: TOOL_DEFINITIONS['create_survey'].name,
+        description: TOOL_DEFINITIONS['create_survey'].description,
+    },
+    {
+        identifier: 'create_feature_flag' as const,
+        name: TOOL_DEFINITIONS['create_feature_flag'].name,
+        description: TOOL_DEFINITIONS['create_feature_flag'].description,
+    },
+    {
+        identifier: 'create_experiment' as const,
+        name: TOOL_DEFINITIONS['create_experiment'].name,
+        description: TOOL_DEFINITIONS['create_experiment'].description,
+    },
```

Comment:
These should be at the start of the list, so that they show up _later_ in the list of tools, after more basic tools like "Query data" (see frontend/__snapshots__/scenes-app-sidepanels--side-panel-max--light.png diff)